content/en/tracing/setup_overview/setup/php.md

```diff
@@ -451,6 +451,24 @@ If no core dump was generated, check the following configurations and change the
 1. Ensure you have a suitable `ulimit` set in your system. You can set it to unlimited: `ulimit -c unlimited`.
 1. If your application runs in a Docker container, changes to `/proc/sys/*` have to be done to the host machine. Contact your system administrator to know the options available to you. If you are able to, try recreating the issue in your testing or staging environments.
 
+### Valgrind trace
+
+Preferably, if you want to give us as many hints as possible, try running the application with valgrind (install it from your package manager). This approach will also work in an unprivileged container (unlike core dumps by default).
+
+   - For CLI this is as simple as executing `USE_ZEND_ALLOC=0 valgrind -- php path/to/script.php`.
+   - When running php-fpm execute `USE_ZEND_ALLOC=0 valgrind --trace-children=yes -- php-fpm -F --fpm-config ...`.
+   - When using apache, execute `(. /etc/apache2/envvars; USE_ZEND_ALLOC=0 valgrind --trace-children=yes -- apache2 -X)`.
+
+### Obtaining a strace
+
+Some issues arise due to external influence, in that case it can be valuable to have a `strace` (install it from your package manager). When providing Datadog support a strace, please make sure to include the `-f` option, to follow child processes as well.
```

Comment:
```suggestion
Some issues are caused by external factors, so it can be valuable to have a `strace`. 

Install `strace` with your package manager. When generating a `strace` to send to Datadog Support, ensure you use the `-f` option to follow child processes.
```